spec/acceptance/basic_candlepin_spec.rb

```diff
@@ -28,4 +28,12 @@
     # Test that the least cipher strength is "strong" or "A"
     its(:stdout) { should match(/least strength: (A|strong)/) }
   end
+
+  describe file("/usr/share/tomcat/conf/cert-users.properties") do
+    it { should be_file }
+    it { should be_mode 640 }
+    it { should be_owned_by 'tomcat' }
+    it { should be_grouped_into 'tomcat' }
+    its(:content) { should match /katelloUser=CN=ActiveMQ Artemis Deferred, OU=Artemis, O=ActiveMQ, L=AMQ, ST=AMQ, C=AMQ/ }
```

Comment:
I think we can have an exact match as well (not sure if the newline check is also needed)
```suggestion
    its(:content) { should eq("katelloUser=CN=ActiveMQ Artemis Deferred, OU=Artemis, O=ActiveMQ, L=AMQ, ST=AMQ, C=AMQ\n") }
```